shared/product.go

```diff
@@ -0,0 +1,104 @@
+package shared
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/config"
+)
+
+// GetProduct returns the distro product based on the config file
+func GetProduct() (string, error) {
+	cfgPath, err := EnvDir(".")
+	if err != nil {
+		return "", ReturnLogError("failed to get config path: %v\n", err)
+	}
+
+	cfg, err := config.AddConfigEnv(cfgPath)
+	if err != nil {
+		return "", ReturnLogError("failed to get config: %v\n", err)
+	}
+	if cfg.Product != "k3s" && cfg.Product != "rke2" {
+		return "", ReturnLogError("unknown product")
+	}
+
+	return cfg.Product, nil
+}
+
+// GetProductVersion return the version for a specific distro product
+func GetProductVersion(product string) (string, error) {
+	if product != "rke2" && product != "k3s" {
+		return "", ReturnLogError("unsupported product: %s\n", product)
+	}
+	version, err := getVersion(product + " -v")
```

Comment:
please only missed this getVersion method to move in here also

Reply:
moved 🙇